src/popup/router/components/FungibleTokens/TokensListItem.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <RouterLink
+    v-if="tokenData"
+    :to="{
+      name: 'token-details',
+      params: { contract: tokenData.contract },
+    }"
+  >
+    <div class="token-row">
+      <Avatar class="token-image" :address="tokenData.contract" :src="tokenData.image || null" />
+      <div class="token-info">
+        <div>
```

Comment:
This part of the layout seems overcomplicated

Reply:
I am not sure what is complicated here.